samples/openthread/common/overlay-ot-defaults.conf

```diff
@@ -28,7 +28,6 @@ CONFIG_NET_IPV6_NBR_CACHE=n
 CONFIG_NET_IPV6_MLD=n
 
 # Stack sizes configuration
-CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=1024
 CONFIG_NET_TX_STACK_SIZE=1200
 CONFIG_NET_RX_STACK_SIZE=1500
 CONFIG_SHELL_STACK_SIZE=2048
```

Comment:
For the CLI sample it's required to increase `CONFIG_SHELL_STACK_SIZE` in order to avoid a usage fault with the Settings system after triggering a board reset. `3072` worked well for me. To be determined if needed for other samples.

Reply:
since all of our samples (maybe except ncp) have shell enabled, this should be included for all of them.

Reply:
`CONFIG_SHELL_STACK_SIZE=2560` is working as well.